test/e2e/features/basic.feature

```diff
@@ -74,4 +75,4 @@ Feature: Basic test
         And kubeconfig is cleaned up
         # cleanup
         When executing crc cleanup command succeeds
-        And executing "man -P cat crc" fails
+#        And executing "man -P cat crc" fails
```

Comment:
why those changes?

Reply:
I'm sorry. It looks like I forgot to uncomment it during testing.

Reply:
I think I lost the changes while pushing from another machine. I'm seeing outdated version of the changes.

Reply:
I've updated it. I think it should be okay now.